djangobb_forum/models.py

```diff
@@ -793,6 +810,16 @@ def filter_ham(self):
         pass
 
     @transition(
+        field=state, source=[UNREVIEWED, FILTERED_HAM, MARKED_HAM], target=USER_DELETED,
+        save=True)
+    def filter_user_deleted(self):
+        """
+        Post is not marked spam by akismet, but user has been globally deleted,
+        putting this into the spam dusbin.
```

Comment:
Just a thought, but we could move these into the regular dustbin if we want, and that would avoid cluttering up the spam dustbin with manually deleted things. The undelete function doesn't care where they end up.